storybook/src/components/banner/banner.core-template.ts

```diff
@@ -9,11 +9,7 @@ export const coreBanner: ComponentImplementation<Banner<TemplateResult>> = {
     function bannerTemplate({ status, compact, noIcon, content }) {
       return html`
         <dso-banner status=${status} ?compact=${compact} ?no-icon=${noIcon}>
-          <div class="container">
-            <div class="row">
-              <div class="col-sm-12">${content}</div>
-            </div>
-          </div>
+          <div class="dso-banner-inner">${content}</div>
```

Comment:
Als ik door onze templates zoek kom ik nu alleen in deze `core-template` en de `css-template` een className tegen met `-inner` erin.

Het lijkt erop dat de conventie is om `-container` te gebruiken.

Zou je `dso-banner-inner` willen vervangen door `dso-banner-container`?

Reply:
Hmm, ik gebruik zelf al jaren `-inner` voor dingen die in een component staan, en `-container` voor dingen die om een component heen staan. Zelf vind ik dat een betere weergave van de werkelijkheid.

Reply:
Prima. Laten we het zo. 👍